React/Modules/RCTAsyncLocalStorage.h

```diff
@@ -16,7 +16,7 @@
  *
  * Current implementation stores small values in serialized dictionary and
  * larger values in separate files. Since we use a serial file queue
- * `RKFileQueue`, reading/writing from multiple threads should be perceived as
+ * `RCTFileQueue`, reading/writing from multiple threads should be perceived as
```

Comment:
Hah. i just did a scan for " RK*" and replaced accordingly.  I'll look to update this and the RCTScrollView soon.

Thanks @brentvatne  =)